src/panels/lovelace/common/entity/toggle-entity.js

```diff
@@ -2,6 +2,7 @@ import { STATES_OFF } from '../../../../common/const.js';
 import turnOnOffEntity from './turn-on-off-entity.js';
 
 export default function toggleEntity(hass, entityId) {
-  const turnOn = STATES_OFF.includes(hass.states[entityId].state);
+  const turnOn = entityId.split('.', 1)[0] === 'scene' ?
```

Comment:
I don't think that this is a good idea. `scene` is not toggle-able, so now this function has a weird edge case?

Reply:
you are right